muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/application-panel.scss

```diff
@@ -106,8 +106,8 @@
 .application-panel__actions {
   background: $color-default;
   left: 0;
-  position: fixed;
   right: 0;
+  position: fixed;
```

Comment:
`position` property needs to be placed before `right` property. Hound should have spotted this as properties order needs to be alphabetical.